build_tools/bazel/build_core.sh

```diff
@@ -83,7 +83,7 @@ bazel \
   --bazelrc=build_tools/bazel/iree.bazelrc \
   query \
     //iree/... + //runtime/... + //samples/... + //build_tools/... + \
-    //llvm-external-projects/iree-dialects/... | \
+    //tools/... + //llvm-external-projects/iree-dialects/... | \
```

Comment:
Oof, I predict we'll run into issues of missed tests because of this. We should probably switch to exclusions rather than inclusions